src/sw-custom.js

```diff
@@ -67,6 +67,24 @@ if ("function" === typeof importScripts) {
         ],
       })
     );
+
+    // Notification event listeners  
+    self.addEventListener('notificationclose', event => {
```

Comment:
These event listeners are such a fascinating part of FE tech for me!

Reply:
@sienna-kopf I think you'd really dig how Event Propogation and Event Bubbling work!